src/abstract-classes/browser-plugin.js

```diff
@@ -179,6 +193,26 @@ class BrowserPlugin {
 
         return false;
     }
+
+    /**
+     *
+     * @param {string} dir - absolute path to the directory.
+     * @returns {Promise<void>}
+     */
+    async _ensureDir(dir) {
+        let dirExists;
+
+        try {
+            await fsPromise.access(dir);
+            dirExists = true;
+        } catch (e) {
+            dirExists = false;
+        }
+
+        if (!dirExists) {
+            await fsPromise.mkdir(dir);
+        }
+    }
```

Comment:
This creates a race condition. You should try to make it and catch the error from making it. But maybe instead of `rimraf`, you could use `fs-extra` and it already has an `ensureDir` and `remove` function.